src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -221,7 +222,7 @@ public async Task<ChainedHeaderBlock> BlockMinedAsync(Block block)
         {
             this.logger.LogTrace("({0}:{1})", nameof(block), block.GetHash());
 
-            PartialValidationResult partialValidationResult;
+            ValidationContext validationResult;
```

Comment:
Shouldn't this be called validation context?

Reply:
fixed